### DISCUSSION TITLE
Multiple trigger function in one pyscript file

### DISCUSSION THREAD

**Discussion by phenates** [custom-components/pyscript#339] [Q&A]
Hy everybody,

First thanks for this great HA integration, it's realy more useful and clear than the 'template' system. Just my opinion...

My question is, can we have multiple trigger function (@state_trigger) in the one file ?
It looks like Pyscript only loads the last trigger and not the previous ones.
I don't talk about multiple decorators on the same function.

```python
@state_trigger("input_button.button1")
def button1(**kwargs):
    log.info("------> Button1")

@state_trigger("input_button.button2")
def button2(**kwargs):
    log.info("------> Button2")
```

Thank you.

**Comment by stefanuytterhoeven** [comment]
you can have as many as you want...

**Comment by stefanuytterhoeven** [comment]
do you still have a problem with this?
